packages/sdk-client/src/client-builder/ClientBuilder.ts

```diff
@@ -198,7 +198,12 @@ export default class ClientBuilder {
   withTelemetryMiddleware<T extends TelemetryOptions<T>>(
     options: T
   ): ClientBuilder {
+    const { version } = require('@commercetools/ts-sdk-apm/package.json')
     const { createTelemetryMiddleware, ...rest } = options
+
+    this.withUserAgentMiddleware({
+      customAgent: `typescript-sdk-newrelic-middleware/${version}`,
```

Comment:
Nice! Now it becomes a static string and also uses the composition of the the different agent parts.

Also this may override custom agent name from a user we should go with this for now